monitor/ui/pom.xml

```diff
@@ -26,6 +27,9 @@
 						<goals>
 							<goal>compile</goal>
 						</goals>
+						<configuration>
+							<sourceLevel>1.7</sourceLevel> <!-- 1.6 or 1.7 -->
```

Comment:
Now we use 2.7.0 which is the latest "stable" release. There's 2.8.0-* as beta and release candidates.